src/jimp2/wireworld/z8/wireworldlogic/World.java

```diff
@@ -22,11 +18,17 @@ public void buildTheWorld(int width, int height, List<Element> elements) {
         // TODO implement here
     }
 
-    /**
-     * @param world
-     */
-    public void copyCells(World world) {
-        // TODO implement here
+    public World copyCells() {
+        World worldTo = new World(width,height);
+
+        for (int i = 0; i < width; i++)
+        {
+            for(int j = 0; j < height; j++)
+            {
+                worldTo.cells[i][j].setState(cells[i][j].getState());
+            }
+        }
+        return worldTo;
```

Comment:
Niepotrzebna alokacja obiektu typu World, wydajniej byłoby to zrobić "w miejscu". Jeżeli są ku temu jakieś przeciwwskazania to pisz śmiało